### DISCUSSION TITLE
[Backend] Ditsmod (Node.js framework writen in TypeScript)

### DISCUSSION THREAD

**Discussion by KostyaTretyak** [realworld-apps/realworld#833] [Added implementations]
https://github.com/ditsmod/realworld

**Comment by KostyaTretyak** [comment]
Hey, @geromegrignon. How I can moved this topic to "Added implementations"?

**Reply by geromegrignon** [reply]
Done, thanks for your contribution !

**Comment by KostyaTretyak** [comment]
You can see more info on https://ditsmod.github.io/realworld/en/
